.node/postcss.config.js

```diff
@@ -1,11 +1,13 @@
 module.exports = {
   plugins: [
-    require("postcss-nested"),
-    require("postcss-preset-env")({
+    require('postcss-preset-env')({
       stage: 0,
       features: {
-        "matches-pseudo-class": false,
+        'matches-pseudo-class': false,
```

Comment:
Maybe double quotes here? As long as it's consistent, either are fine.

Reply:
Not sure why my format-on-save didn't pick this up in the first instance...

Anyway, this is fixed by d29dd5f961e382940ce91af829b72a873a396fac.